core/templates/components/question-directives/questions-list/questions-list.directive.ts

```diff
@@ -607,6 +612,7 @@ angular.module('oppia').directive('questionsList', [
             ctrl.associatedSkillSummaries = [];
             ctrl.selectedSkillId = ctrl.getSelectedSkillId();
             ctrl.editorIsOpen = false;
+            ctrl.questionIdDeleted = 'yyyy';
```

Comment:
er.. what's 'yyyy'?

Why can't we use a boolean flag called `questionIsBeingDeleted` or similar?

Reply:
I tried using that , in that case we will set it back to false as the function execution is over so that we can delete the next question. Again the UI takes a bit more time to update than the function execution and the same problem was persisting.

Reply:
You will need to disable the button using this flag. i.e. when a deletion is in progress, you can't click it again.